charts/firecloudui/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: firecloudui
-version: 0.1.0
+version: 0.2.0
```

Comment:
The version will be auto-bumped once you merge, so no need to bump manually

Reply:
Thanks; I forget why we did this during the PoC config changes, but there was a reason.

Reply:
probably to use the `>~` fuzzy version match syntax against the pr chart releases while testing

Reply:
Reverted to `0.1.0`.